src/passes/Inlining.cpp

```diff
@@ -552,7 +552,7 @@ struct FunctionSplitter {
   // into account (like limitations on which functions can be inlined into in
   // each iteration, the number of iterations, etc.). Therefore this function
   // may only find out if we *can* split, but not actually do any splitting.
-  InliningMode getSplitInliningMode(Function* func) {
+  InliningMode getSplitDrivenInliningMode(Function* func, FunctionInfo& info) {
```

Comment:
Please add/update the comment above here, as now the function can return `Full` as well.